testkit/testkit.go

```diff
@@ -114,17 +119,19 @@ func (tk *TestKit) Session() session.Session {
 
 // MustExec executes a sql statement and asserts nil error.
 func (tk *TestKit) MustExec(sql string, args ...interface{}) {
-	defer func() {
-		if tk.alloc != nil {
-			tk.alloc.Reset()
-		}
-	}()
 	tk.MustExecWithContext(context.Background(), sql, args...)
 }
 
 // MustExecWithContext executes a sql statement and asserts nil error.
 func (tk *TestKit) MustExecWithContext(ctx context.Context, sql string, args ...interface{}) {
```

Comment:
Considering all functions will eventually invoke `MustExecWithContext`, why adding hooks to every exec and query function?

Reply:
Cannot get all the error information in the MustExecWithContext function，such as GetRows4Test.I need to get the final result for exception handling, so I can't handle it in MustExecWithContext